flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/RemoteInputChannel.java

```diff
@@ -70,6 +78,15 @@
 	 */
 	private int expectedSequenceNumber = 0;
 
+	/** The initial number of exclusive buffers assigned to this channel. */
+	private int initialCredit;
+
+	/** The current available buffers including both exclusive buffers and requested floating buffers. */
+	private final ArrayDeque<Buffer> availableBuffers = new ArrayDeque<>();
+
+	/** The number of available buffers that have not been unannounced to producer yet. */
```

Comment:
actually, there were some more typos here - the full comment should be something like this `The number of available buffers that have not been announced to the producer yet.`